docs/rules/no-warning-comments.md

```diff
@@ -109,10 +109,10 @@ As mentioned above, patterns are matched when they match exactly to one of the t
    ...
    ```
 
-5. Rule configured to warn on matches of the specified terms at the start of comments. Note that you can use as many terms as you want.
+5. Rule configured to warn on matches of the specified terms at any location in the comments. Note that you can use as many terms as you want.
 
    ```js
    ...
-   "no-warning-comments": [1, { "terms": ["any really", "interesting", "or even not", "term", "can be matched"] }]
+   "no-warning-comments": [1, { "terms": ["really any", "term", "can be matched"], "location": "anywhere" }]
```

Comment:
Changed this so that one of the examples showed use of both options (changing "location" as well as "terms").